### PR TITLE
coredump: initialize copy array

### PR DESCRIPTION
**Pull request by sshedi**
Fixes: CID#1469723

### CODE REVIEW

File: src/coredump/coredump.c [hunk 1]
```diff
@@ -1074,7 +1074,8 @@ static int send_iovec(const struct iovec_wrapper *iovw, int input_fd) {
                         .msg_iov = iovw->iovec + i,
                         .msg_iovlen = 1,
                 };
-                struct iovec copy[2];
+
+                struct iovec copy[2] = {0};
```
Comment (yuwata) [inline src/coredump/coredump.c]: Note, we use `{}` instead of `{0}`.

**Review by yuwata — CHANGES_REQUESTED** [review]
This also sounds false positive for me...

File: src/coredump/coredump.c (hunk 1 above)
Comment (sshedi) [reply src/coredump/coredump.c]: I saw that but I think Coverity is not happy with it so decided to use `{0}`, I will fix it. Thanks.

File: src/coredump/coredump.c [hunk 2]
```diff
@@ -1074,7 +1074,8 @@ static int send_iovec(const struct iovec_wrapper *iovw, int input_fd) {
                         .msg_iov = iovw->iovec + i,
                         .msg_iovlen = 1,
                 };
-                struct iovec copy[2];
+
+                struct iovec copy[2] = { };
```
Comment (yuwata) [inline src/coredump/coredump.c]: We do not use `{ }`. Please use `{}`.

**Review by yuwata — CHANGES_REQUESTED** [review]
Still I do not understand the issue. Please explain more.

**Comment by sshedi** [comment]
> Still I do not understand the issue. Please explain more.
Again, this is a false positive and an attempt to keep coverity quiet.

**Pull request closed without merging**